client/api_handler.py

```diff
@@ -58,6 +62,13 @@ def get_metadata(url, response_type=ResponseType.JSON):
             return get_json_metadata(metadata.data_map)
 
     metadata.parse_content(site_response)
+
+    # Trim the description if necessary
+    if metadata.prop_map['description'] is not None:
```

Comment:
actually maybe it makes more sense to just keep it in this rather than put it into metadata function. I'd just extract it out into it's own function